source/n_fields_problem.cc

```diff
@@ -109,8 +109,34 @@ declare_parameters (ParameterHandler &prm)
                   "true",
                   Patterns::Bool());
 
```

Comment:
You don't need any of the following if you use parsed grid refinement... all of these are included there!